DevProject/Packages/manifest.json

```diff
@@ -10,7 +10,7 @@
     "com.unity.ide.vscode": "1.1.4",
     "com.unity.ml-agents": "file:../../com.unity.ml-agents",
     "com.unity.ml-agents.extensions": "file:../../com.unity.ml-agents.extensions",
-    "com.unity.multiplayer-hlapi": "1.0.4",
+    "com.unity.multiplayer-hlapi": "1.0.6",
```

Comment:
This was producing errors:
```
[20:53:43.945 INF] Compiler
	severity: Error
	message: Deterministic compilation failed. You can disable Deterministic builds in Player Settings
Library/PackageCache/com.unity.multiplayer-hlapi@1.0.4/Editor/Tools/Weaver/AssemblyInfo.cs(22,28): error CS8357: The specified version string contains wildcards, which are not compatible with determinism. Either remove wildcards from the version string, or disable determinism for this compilation
	stacktrace: 
	line: 22
	file: Library/PackageCache/com.unity.multiplayer-hlapi@1.0.4/Editor/Tools/Weaver/AssemblyInfo.cs
```
Should be fixed in https://docs.unity3d.com/Packages/com.unity.multiplayer-hlapi@1.0/changelog/CHANGELOG.html#106---2020-05-08